host/sysfs/gpio.go

```diff
@@ -167,19 +167,22 @@ func (p *Pin) In(pull gpio.Pull, edge gpio.Edge) error {
 				return p.wrap(err)
 			}
 		}
-		if p.edge != edge {
-			var b []byte
-			switch edge {
-			case gpio.RisingEdge:
-				b = bRising
-			case gpio.FallingEdge:
-				b = bFalling
-			case gpio.BothEdges:
-				b = bBoth
-			}
-			if err := seekWrite(p.fEdge, b); err != nil {
-				return p.wrap(err)
-			}
+		// Always reset the edge detection mode to none after starting the epoll,
+		// otherwise edges are not always delivered.
```

Comment:
Can you expand the comment like "as observed on a Allwinner A20 running kernel x.y.z". Otherwise we'll have no clue why this was added.

Reply:
Good thinking, done!